_posts/2020-08-05-security-of-nano.md

```diff
@@ -0,0 +1,81 @@
+![](/images/ledger_x.jpeg)
+
+### Keys on Paper
+This is arguable the safest method since a paper is hopefully offline and hidden from prying eyes but also arguably LESS safe than a hardware wallet because if anyone would find the paper it's also directly accessible and unencrypted. The safety relies on where you hid it. A seed on a paper can be scanned by Nault and used when you need access to it, either doing an online signing or offline signing.
+![](/images/security_02.png)
+
+============================
+## The usage of Keys
+The third factor that decides the overall security of your Nano is how you choose to transact it. Using Nault is very secure in itself because no sensitive data is sent over the Internet. Nault connects to a backend server where commands are sent to the Nano network, but the block signing is done on your device using keys on your device. Only the final signed block is sent over the Internet but the signature can't be reversed or tampered with. However, what will reflect the security is the key storage method described in the previous chapter.
+
+### Transact with an online Nault
+Your keys are stored encrypted in the browser storage and you unlock the wallet/ decrypt the keys before doing any transactions. This is the most convenient method but not the most secure. Worst case, the wallet is hacked and you are tricked to enter your seed.
+![](/images/security_11.png)
+
+### Transact with the Ledger hardware wallet
+Your keys are secured by the Ledger device and unlocked before doing any transactions. A difference from the native Nault is you have to validate the destination and balance on the device display and manually approve with the device button. A hacked wallet can't trick you in this case because what's shown on the device is always true. This is medium convenient but not even this is the most secure option. Worst case, an unknown vulnerability exists in the device that tricks you into sending funds to someone else.
+
+### Transact using a remote-signing procedure
+Your keys are stored offline, thus avoiding any leaks to the Internet due to malware, hacked password manager or other things that can go wrong in any online device. The principal follows:
+
+1. Create an unsigned block for account X (using the latest data from the network)
+1. Transfer it to an offline device where it's signed using a key that owns X
```

Comment:
Enumeration 1. 1. 1.? Probably rendered correctly anyway

Reply:
It's correct and much easier to edit and less errors